### ISSUE TITLE
Space missing after comma *after* string interpolation

### ISSUE THREAD

**Issue by jvf** [rrrene/credo#492]
### Environment

* Credo version (`mix credo -v`): 0.9.0-rc2
* Erlang/Elixir version (`elixir -v`): Elixir 1.6.0 (compiled with OTP 20)
* Operating system: macOS 10.13.2

### What were you trying to do?

Run `mix credo --strict` on the following code examples:

Example 1:
```elixir
defmodule SpaceMissing1 do
  @moduledoc false

  @doc false
  def test_credo do
    foo = 1
    bar = 2
    _ = "#{foo} #{bar}"
    _ = Enum.join([1, 2, 3], ",") # comma in the second argument is reported
    _ = ~r"," # comma is reported
  end
end
```
### Expected outcome

That no errors are reported.

### Actual outcome

```elixir
┃ [R] → Space missing after comma
┃       lib/space_missing_1.ex:10:12 #(SpaceMissing1.test_credo)
┃ [R] → Space missing after comma
┃       lib/space_missing_1.ex:9:31 #(SpaceMissing1.test_credo)
```

**Comment by rrrene** [comment]
Thanks for reporting this 😀  It is now fixed on `master`.

You can try this by setting the Credo dep to

```elixir
{:credo, github: "rrrene/credo"}
```

Please report back if your issue is solved! 👍

**Comment by jvf** [comment]
Issue is solved, checked against my original code as well as against the minimal example above.
